ui/pages/settings/networks-tab/index.scss

```
@@ -1,10 +1,50 @@
+@use "design-system";
+
+p#search-networks-helper-text {
+  color: var(--color-text-alternative);
+}
+
+.imageclose {
+  cursor: pointer;
+  color: var(--color-icon-default);
+}
+
 .networks-tab {
   &__content {
     display: flex;
     height: 100%;
-    max-width: 739px;
+    max-width: 779px;
     justify-content: space-between;
 
+    &__custom-image {
+      border: 1px solid var(--color-border-default);
+      margin-inline-start: 8px;
+    }
+
+    &__check-icon {
+      margin-inline-end: 10px;
+      color: var(--color-success-default);
+
+      &__transparent {
+        color: transparent;
+        width: 16px;
+        margin-inline-end: 10px;
+      }
+    }
+
+    &__icon-with-fallback {
+      padding: 0 1px 2px 2px;
+      color: var(--white);
```

We are currently removing all instances of the old css variables. `primary-inverse` which is the color that should be used as for any foreground elements on top of our primary color will work for now. I'll make a note to update in future.

```suggestion
      color: var(--color-primary-inverse); // TODO: design-tokens needs network colors
```

---

@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/13560/commits/c48e1890932565addc62ccd37f3dfad1a324c708)